ga4gh/frontend.py

```diff
@@ -41,49 +45,64 @@ def handleHTTPOptions():
     return response
 
 
+@app.errorhandler(Exception)
+def handle_error(error):
+    if not isinstance(error, exceptions.ParentException):
+        if app.config['DEBUG']:
+            print(traceback.format_exc(error))
+        error = exceptions.ServerException()
+
+    protocolException = protocol.GAException()
+    protocolException.errorCode = error.code
+    protocolException.message = error.message
+    response = flask.jsonify(protocolException.toJSONDict())
```

Comment:
We should set the content-type here. In the long run, we may want to offer other types.